components/layouts/page-card.tsx

```diff
@@ -11,14 +13,15 @@ function PageCard({ children }: { children: React.ReactNode }) {
         minHeight: '100vh',
       }}
     >
-      <Header />
+      <Guide />
```

Comment:
I'm noticing some grid alignment issues, both on the index page and the feedstock page:

|  |  |
| - | - |
|  ![CleanShot 2024-10-03 at 16 58 31@2x](https://github.com/user-attachments/assets/21011d0c-e2d4-4c20-80b5-eb96e77cba96) | Is it possible to make the feedstock cards take extend to take up the full four columns? (i.e. occupy the margins highlighted in yellow) |
| ![CleanShot 2024-10-03 at 17 00 18@2x](https://github.com/user-attachments/assets/a7e477d2-3d47-4581-9f3d-84342d25b7ca) | Is it possible align the various metadata fields to the grid (license, repository, data viewer link, etc.)? |